packages/transformers/svg-react/src/SVGRTransformer.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import JSTransformer from '@parcel/transformer-js';
+
+import SVGO from 'svgo';
+import path from 'path';
+import camelcase from 'camelcase';
+import svgToJsx from 'svg-to-jsx';
+
+import * as babelParser from '@babel/parser';
+
+const defaultConfig = {
+  plugins: [{prefixIds: true}],
+};
+
+function getComponentName(filePath) {
+  let validCharacters = /[^a-zA-Z0-9_-]/g;
+  let name = path.parse(filePath).name.replace(validCharacters, '');
+  let pascalCaseFileName = camelcase(name, {
+    pascalCase: true,
+  });
+
+  return `Svg${pascalCaseFileName}`;
+}
+
+const [parcelPluginSymbol] = Object.getOwnPropertySymbols(JSTransformer);
+
+JSTransformer[parcelPluginSymbol].getConfig = async function({asset}) {
```

Comment:
@DeMoorJasper - My goal was to re-use the logic in `JSTransformer`, since ideally the only difference is this transformer does a "pre-transform" to get it in a react component state. Other than that, it should go through the normal JS transformation. The reason I wanted to do this was to preserve the babel transforms, since I know differential bundling is a key feature of parcel 2.

Is what I said above accurate? If so, is the right approach to somehow extend `JSTransformer`? If not, should I just duplicate the necessary logic from `JSTransformer` in this one?

Reply:
In Parcel 2 we have a concept called pipelines so in theory there would be no need to extend any transformer as you can let another transformer run before or after this transformer

Reply:
@DeMoorJasper - I updated the transformer to just generate the react code and not run any JS transforms. In my application, I can then update my `.parcelrc` to run it through both transformers:

```json
{
  "extends": ["@parcel/config-default"],
  "transforms": {
    "*.svg": ["@parcel/transformer-svgo", "@parcel/transformer-svg-react", "@parcel/transformer-js"]
  }
}
```

Is there any way to specify a "follow-up" transformer? For SVG to react, it seems to make sense to always run the result of this transformer through the one that handles JS, so it is transformed for the target environment properly. Thoughts?